tests/main/try-non-fatal/task.yaml

```diff
@@ -0,0 +1,18 @@
+summary: Checks that removing the base directory of a tried snap works.
+
+execute: |
+    echo "Given a tried snap"
+    base_dir=$(mktemp -d)
+    cp -R $TESTSLIB/snaps/test-snapd-tools/* $base_dir
```

Comment:
nitpick^2: maybe `cp -a` is preferable but it does not matter for this test (because -a will keep symlinks symlinks but we don't use any in the dir so…)